Libraries/Core/Timers/JSTimers.js

```diff
@@ -494,4 +495,7 @@ if (!Timing) {
   ExportedJSTimers = JSTimers;
 }
 
+BatchedBridge.setImmediatesCallback(
+  ExportedJSTimers.callImmediates.bind(ExportedJSTimers));
```

Comment:
Can you make sure to run Prettier over this file? I'm pretty sure this will be formatted with `);` on the next line.